inc/classes/subscriber/Optimization/class-minify-js-subscriber.php

```diff
@@ -1,10 +1,10 @@
 <?php
```

Comment:
Let's move and rename this class file too.

Reply:
@Tabrisrp This file still needs to be moved and renamed.

Reply:
Same here, in the files tab, it's correctly showing as updated